src/codelabs/null-safety.md

```diff
@@ -427,19 +432,20 @@ For cases like that, use the
 [`late` keyword](/null-safety/understanding-null-safety#late-variables).
 
 When you put `late` in front of a variable declaration,
-that tells Dart the following:
+that tells Dart the following about the variable:
 
--   Don't assign that variable a value yet.
--   You *will* assign it a value later.
--   You'll make sure that the variable has a value *before* the variable is used.
+- It cannot be assigned a value yet.
```

Comment:
This is too strong. I might be that it could be assigned a value, but that the code for that would just be too cumbersome.

`late` is the developer expressing *their* intent: "I don't want to assign assign value just yet"